content/reference/sql/version-control/merges.md

```diff
@@ -0,0 +1,158 @@
+| NULL            | NULL           |    NULL | NULL            |     NULL | Student        | Simpson       |      3 | Bart           |      10 | Student          | Simpson         |        3 | Lisa             |         8 |
++-----------------+----------------+---------+-----------------+----------+----------------+---------------+--------+----------------+---------+------------------+-----------------+----------+------------------+-----------+
+```
+
+For each column in the database table, the `conflicts` table has three
+columns: one for `base`, one for `ours` and one for `theirs`. These
+represent the three different values you might choose to resolve the
+conflict (either the common commit ancestor's values, the current
+table values, or the merged table values).
+
+## Resolving conflicts
+
+To commit your transaction, you must first resolve the merge conflicts
+by deleting every row in every `dolt_conflicts` system table. This
+signals to Dolt that you have resolved every merge conflict to your
+satisfaction. There are several different strategies you could use,
+which you must repeat for every table in conflict.
+
+### Take ours
+
+To use the values in the current working set (rather than the merged
+values), simply delete from the `dolt_conflicts` table without
+changing anything else.
+
+```sql
+DELETE FROM dolt_conflicts_people;
+```
+
+### Take theirs
+
+To use the merged values, overwriting our own, `REPLACE` and `DELETE`
+rows from the table using the conflicts table:
+
+```sql
+-- Replace existing rows with rows taken with their_* values as long 
+-- as their_id is not null (rows deleted in theirs)
+REPLACE INTO people (id,first_name,last_name,age) (
+    SELECT their_id, their_first_name, their_last_name, their_age
+    FROM dolt_conflicts_people
+    WHERE their_id IS NOT NULL
+);
+
+-- Delete any rows that are deleted in theirs
+DELETE FROM PEOPLE WHERE id IN (
+    SELECT base_id
+    FROM dolt_conflicts
+    WHERE base_id IS NOT NULL AND their_id IS NULL
+);
+
+-- mark conflicts resolved
+DELETE FROM dolt_conflicts_people;
+```
+
+### Custom logic
+
+It's also possible that you want your users to resolve conflicts
+themselves by picking which of the conflicting values to use for each
+row in conflict. You can build this workflow, or any other custom
+logic you want, with the SQL primitives above.
+
+## Committing with merge conflicts
+
+By default, Dolt will not allow you to commit transactions that have
+merge conflicts, and will automatically rollback any transaction with
+merge conflicts when you attempt to `COMMIT` it. However, there may be
+times when you need to commit merge conflicts, for example to
+collaborate on resolving them with other people. To allow committing
+merge conflicts, change this system variable to `1` for every client
+that needs to commit merge conflicts:
+
+```sql
+set @@dolt_allow_commit_conflicts = 1;
+```
+
+The server will not allow you to create new Dolt commits (with the
```

Comment:
Something that may be important to clarify is that the moment you sql transaction commit, commit conflicts into a working set, any other MySQL client has to set `@@dolt_allow_commit_conflicts = 1` even if they aren't producing new conflicts.

Reply:
what happens if they don't?

Reply:
Feel free to add this note yourself.